Makefile

```diff
@@ -53,6 +53,7 @@ help: ## Display this help.
 .PHONY: manifests
 manifests: ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	@rm -rf config/crd/bases/_.yaml
```

Comment:
Q: why is this necessary?  This file doesn't exist for me when I build from a clean repository.

Reply:
I think it's a bug in controller-gen triggered by the PodSpecable duck type resource which is like an object but shouldn't generate a CRD. I can try to hunt the origin another time, nuking the output seemed like the easiest path forward.